src/components/Messages/index.vue

```diff
@@ -48,9 +53,16 @@
             content: newValue,
           };
 
-          this.$http.post(this.apiURL, params).then(() => {
-            this.getAllMessages();
-            this.newMessage = '';
+          this.$http.post(this.apiURL, params)
```

Comment:
this.$http.post(this.apiURL, params)
.catch(() => IndexDB.saveMessageOffline(this.pseudo, newValue))
.then(() => 
     this.getAllMessages();
     this.newMessage = '';
))

Reply:
juste pour l'idée